pkg/karmadactl/promote/promote.go

```diff
@@ -681,7 +681,7 @@ func (o *CommandPromoteOption) createPropagationPolicy(karmadaClient *karmadacli
 }
 
 // createClusterPropagationPolicy create ClusterPropagationPolicy in karmada control plane
-func (o *CommandPromoteOption) createClusterPropagationPolicy(karmadaClient *karmadaclientset.Clientset, obj *unstructured.Unstructured, gvr schema.GroupVersionResource, isDep bool) error {
+func (o *CommandPromoteOption) createClusterPropagationPolicy(karmadaClient *karmadaclientset.Clientset, _ *unstructured.Unstructured, gvr schema.GroupVersionResource, isDep bool) error {
```

Comment:
Why not remove the parameter?

Reply:
Do you mean removing all of the unused parameters or to remove the parameter of this specific function only?

Reply:
I mean like this:
```diff
@@ -681,7 +681,7 @@ func (o *CommandPromoteOption) createPropagationPolicy(karmadaClient *karmadacli
 }

 // createClusterPropagationPolicy create ClusterPropagationPolicy in karmada control plane
-func (o *CommandPromoteOption) createClusterPropagationPolicy(karmadaClient *karmadaclientset.Clientset, _ *unstructured.Unstructured, gvr schema.GroupVersionResource, isDep bool) error {
+func (o *CommandPromoteOption) createClusterPropagationPolicy(karmadaClient *karmadaclientset.Clientset, gvr schema.GroupVersionResource, isDep bool) error {
        var policyName string
        if o.PolicyName == "" {
                policyName = names.GeneratePolicyName("", o.name, o.gvk.String())
```